### PR TITLE
fix: Auto teleport tooltip

### LINKED ISSUE #8451: Auto-teleport in confirm modal simplified follow-up
not sure if related, but in the case of the not funds, when hovering over the tooltip, its not overflow and its makes the modal horizontal scrollable 

image

cc @hassnian

_Originally posted by @exezbcz in https://github.com/kodadot/nft-gallery/issues/8375#issuecomment-1837549399_

### LINKED ISSUE #8820: Auto teleport tooltip
- we should make the tooltip more ux friendly 
- currently it shows on hover, but for it to close, you have to click elsewhere 

can we make it work similarly as the collection tooltip 
image

## Styling 
- please see the spacing inside figma 
- mainly the vertical
- https://www.figma.com/file/JXNtryaNYhY0XJNArzvPQx/Handoff?type=design&node-id=4354-15507&mode=design&t=fIRXZQFq7DOdhU7s-11
image
image

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- use tippy instate of NeoTooltip
- fixed vertical margins
- renamed file from AutotelportTooltip to AutotelportPopever

- [x] Closes #8820 
- [x] Closes #8451

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

CleanShot 2024-01-02 at 13 37 52

CleanShot 2024-01-02 at 13 40 13

### CODE REVIEW

**Comment by hassnian** [comment]
@exezbcz please review

**Review by roiLeo — COMMENTED** [review]
is `v-tippy` better than `NeoTooltip`? 
should we use only one of the tooltip component?

**Comment by hassnian** [comment]
> is `v-tippy` better than `NeoTooltip`? should we use only one of the tooltip component?

first time using v-tippy , so idk , seems to be better in some aspects but we are only using it in 3 places 

The only reason I ended up using v-tippy it's because I couldn't get the same behaviour using the oruga component. 

using only one would good  idea though

**Comment by exezbcz** [comment]
yup looks good thanks! 


one issue i came across is this now. Not sure if related to the deploy, but this does not seem right 


image
image
image


seems like its also a canary issue

the autoteleport should not be active 

my address: 
14JipEfjkyFEe4jRViuqFomLPBz96SEXKGcYVkCcfqbAqAxU
nft: 
https://canary.kodadot.xyz/ahp/gallery/50-533

**Comment by hassnian** [comment]
> yup looks good thanks!

@exezbcz can you add the visual ok tag 

thanks ;D

**Review by exezbcz — APPROVED** [review]

File: components/common/autoTeleport/AutoTeleportPopover.vue [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+</script>
+<style lang="scss" scoped>
+@import '@/assets/styles/abstracts/variables';
+
+.popover-container {
+  max-width: 16rem;
+
+  @include ktheme() {
+    background: theme('background-color');
+  }
```
Comment (Jarsen136) [inline components/common/autoTeleport/AutoTeleportPopover.vue:83]: Please check `theme-background-color` at `global.scss`

Comment (yangwao) [reply components/common/autoTeleport/AutoTeleportPopover.vue:83]: we can do follow up

**Comment by yangwao** [comment]
Thanks!
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 8.58 USD/DOT ~ 3.497 $DOT
🧗 [13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh](https://polkadot.subscan.io/account/13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)
🔗 [0xb96543f954d0b07062092e7f3bfdb7b86fa852db0d12e6973ec4f3abadc9b0af](https://polkadot.subscan.io/extrinsic/0xb96543f954d0b07062092e7f3bfdb7b86fa852db0d12e6973ec4f3abadc9b0af)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**